control_ext.go

```diff
@@ -0,0 +1,36 @@
+package ldap
+
+import (
+	ber "github.com/go-asn1-ber/asn1-ber"
+)
+
+// ControlMicrosoftServerExtendedDnOid implements the control LDAP_SERVER_EXTENDED_DN_OID described in
+// https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-adts/3c5e87db-4728-4f29-b164-01dd7d7391ea
+type ControlMicrosoftServerExtendedDnOid struct {
```

Comment:
Please implement the `New` function for this to maintain consistency, although it may seem superfluous.
For example, see [here](https://github.com/go-ldap/ldap/blob/d53d0d99860eaa6a14a5bedc9879911c5621ff3e/v3/control.go#L275)